packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardWithFilters.tsx

```diff
@@ -118,28 +119,52 @@ export const CloudPulseDashboardWithFilters = React.memo(
     });
 
     return (
-      <Box display={'flex'} flexDirection={'column'} gap={2.5}>
-        <Box display={'flex'} flexDirection={'column'} gap={1}>
-          <Box alignSelf="end" width={160}>
-            <CloudPulseTimeRangeSelect
-              disabled={!dashboard}
-              handleStatsChange={handleTimeRangeChange}
-              hideLabel
-              savePreferences={true}
-            />
-          </Box>
-
-          <Paper
-            sx={{
-              padding: 0,
-            }}
-          >
+      <Box display="flex" flexDirection="column" gap={2.5}>
+        <Paper
+          sx={{
+            padding: 0,
+          }}
+        >
+          <Grid container>
+            <Grid container item m={3} rowGap={1} xs={12}>
+              <Grid
+                columnSpacing={2}
+                container
+                item
+                justifyContent="space-between"
+                rowSpacing={2}
+              >
+                <Grid display={'flex'} item md={4} sm={5} xs={12}>
+                  <CloudPulseDashboardSelect
+                    defaultValue={dashboardId}
+                    disabled
+                    serviceIntegration
+                  />
+                </Grid>
+                <Grid display="flex" gap={1} item md={4} sm={5} xs={12}>
```

Comment:
do we need gap here, just asking

Reply:
I've taken this complete structure from global filters. I've to see if it is required and need to modify in global filters as well